src/youtube-to-tana.tsx

```
@@ -141,12 +141,53 @@ async function getYouTubeTab(): Promise<{ url: string; tabId: number; title?: st
+          format: 'text',
+          cssSelector: 'title',
+        }),
+        3000,
+        'Getting focused tab title',
+      )
+
+      console.log(`🔍 Focused tab title: "${focusedTabTitle}"`)
+
+      // Check if focused tab is a YouTube video
+      if (focusedTabTitle && focusedTabTitle.includes('YouTube')) {
+        // Find the tab that matches our focused tab title
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)